pkgs/by-name/pr/proxyman/package.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  appimageTools,
+  fetchurl,
+  asar,
+}:
+let
+  pname = "proxyman";
+  version = "2.25.0";
+
+  src = fetchurl {
+    name = "${pname}-${version}.AppImage";
+    url = "https://download.proxyman.com/linux/${version}/build/Proxyman-${version}.AppImage";
+    hash = "sha256-vS0cijabjTPonJkT5eMfamKbMfgsJV8SlDK1QbnR/Pg=";
+  };
+
+  appimageContents = (appimageTools.extract { inherit pname version src; }).overrideAttrs (oA: {
+    buildCommand = ''
+      ${oA.buildCommand}
+      ${asar}/bin/asar extract $out/resources/app.asar app
+
+      # This will fix the issue with Proxyman not detecting NixOS as a valid Linux environment
+      substituteInPlace app/dist/main/main.js --replace-fail "/etc/ca-certificates/trust-source/anchors/" "/etc/ssl/certs/"
+
+      # This will permanently mark the certificate as installed, as this should be done through Nix config rather than
+      # placing / editing a file in /etc like Proxyman would expect.
+      # Configure the certificate located in "~/.config/Proxyman/certificate/certs/ca.pem" using security.pki.certificates in your nix config
+      substituteInPlace app/dist/main/main.js --replace-fail "return this.isFile(this.getNewCertPath(e))" "return true"
+
+      ${asar}/bin/asar pack app $out/resources/app.asar
+    '';
+  });
+
+in
+appimageTools.wrapAppImage {
+  inherit pname version;
+  src = appimageContents;
+
+  extraInstallCommands = ''
+    install -Dm444 ${appimageContents}/proxyman.desktop -t $out/share/applications
+    install -Dm444 ${appimageContents}/proxyman.png -t $out/share/pixmaps
+    substituteInPlace $out/share/applications/proxyman.desktop \
+      --replace 'Exec=AppRun' "Exec=$out/bin/${pname} --"
+  '';
+
+  meta = {
+    description = "Proxyman enables developers to capture, inspect, and manipulate HTTP(s) requests/responses with ease.";
```

```suggestion
    description = "Capture, inspect, and manipulate HTTP(s) requests/responses with ease";
```

https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes